label_sync/labels.md

```diff
@@ -25,15 +25,16 @@ larger set of contributors to apply/remove them.
 
 | Name | Description | Added By | Prow Plugin |
 | ---- | ----------- | -------- | --- |
-| <a id="triage/duplicate" href="#triage/duplicate">`triage/duplicate`</a> | Indicates an issue is a duplicate of other open issue. The usage should be supported by adding triage findings as an issue comment. <br><br> This was previously `close/duplicate`, | humans | |
+| <a id="triage/duplicate" href="#triage/duplicate">`triage/duplicate`</a> | Indicates an issue is a duplicate of other open issue. The usage should be supported by adding triage findings as an issue comment. <br><br> This was previously `close/duplicate`, `duplicate`, | humans | |
 | <a id="triage/needs-information" href="#triage/needs-information">`triage/needs-information`</a> | Indicates an issue needs more information in order to work on it. The usage should be supported by adding triage findings as an issue comment. <br><br> This was previously `close/needs-information`, | humans | |
 | <a id="triage/not-reproducible" href="#triage/not-reproducible">`triage/not-reproducible`</a> | Indicates an issue can not be reproduced as described. The usage should be supported by adding triage findings as an issue comment. <br><br> This was previously `close/not-reproducible`, | humans | |
-| <a id="triage/unresolved" href="#triage/unresolved">`triage/unresolved`</a> | Indicates an issue that can not be resolved. The usage should be supported by adding triage findings as an issue comment. <br><br> This was previously `close/unresolved`, | humans | |
+| <a id="triage/support" href="#triage/support">`triage/support`</a> | Indicates an issue that is an support question that was opened incorrectly. The usage should be supported by adding triage findings as an issue comment. <br><br> This was previously `close/support`, `kind/support`, `question`, | humans | |
```

Comment:
Nit - replace "an support question" to "a support question" ? OR change the sentence to "Indicates an issue is a support question that was opened incorrectly."? that will be more aligned with other sentences (e.g. LOC 28 above)